src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -39,32 +84,59 @@ public void close() {
     }
   }
 
-  public void activateLeader(MotorMode m) {
-    motor.set(m.dutyCycle);
+  public void activateMotors(MotorMode m) {
+    leftMotor.set(m.dutyCycle);
+    rightMotor.set(m.dutyCycle);
   }
 
   public void openAngler() {
-    anglerPiston.set(DoubleSolenoid.Value.kForward);
+    piston.set(DoubleSolenoid.Value.kForward);
   }
 
   public void closeAngler() {
-    anglerPiston.set(DoubleSolenoid.Value.kReverse);
+    piston.set(DoubleSolenoid.Value.kReverse);
+  }
+
+  private Command reachSideHeight(
+      SparkMaxPIDController controller, RelativeEncoder encoder, double height) {
+    return new TrapezoidProfileCommand(
+            new TrapezoidProfile(
```

Comment:
I think that the profiles can be created on construction and cached, no need to recreate them every invocation.